nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/controller-service/controller-service-table/controller-service-table.component.ts

```diff
@@ -208,7 +208,7 @@ export class ControllerServiceTable implements AfterViewInit {
     }
 
     canConfigure(entity: ControllerServiceEntity): boolean {
-        return this.canRead(entity) && this.canWrite(entity) && this.isDisabled(entity);
+        return this.canRead(entity) && this.canWrite(entity);
```

Comment:
In current NiFi the configure action button on each row is present even when disabled. @mcgilman is this behavior correct?

Reply:
Current NiFi UI does allow the configuration dialog to open when the component is Enabled. This was introduced in an effort that removed the read-only specific dialog and updated the configuration dialog to support both read-only and configuration modes. The new UI currently does not support a read-only dialog from within the configuration dialog. We do have a line item in NIFI-12400 to introduce this behavior. Once the behavior is in place, we would want to remove the disabled condition here.

Reply:
We should leave this in until our configuration dialog supports a read-only mode for viewing configuration when the component is Running or Enabled.